app/questionPage.html

```diff
@@ -89,7 +128,7 @@ <h4 class="timer"></h4>
 <div id="overview_page">
     <div class="container">
         <div class="row">
-            <img class="logo" alt="iO Academy Logo" src="./images/logos/SVGs/iO_Logo_Alt.svg">
+                <img class="logo" alt="iO Academy Logo" src="./images/logos/SVGs/iO_Logo_Alt.svg">
```

Comment:
random added indent? i think it was correct without it

Reply:
fixed